composer.json

```diff
@@ -17,6 +17,7 @@
     "autoload": {
         "psr-4": {
             "React\\EventLoop\\": "src"
-        }
+        },
+        "files": ["src/functions.php"]
```

Comment:
Personally I'm fan of using an intermediary `functions_include.php` like this, this avoids weird edge cases when using threads:
* https://github.com/WyriHaximus/TickingPromise/blob/master/composer.json#L25
* https://github.com/WyriHaximus/TickingPromise/blob/master/src/functions_include.php